src/Illuminate/Mail/resources/views/text/message.blade.php

```diff
@@ -1,27 +1,27 @@
-@component('mail::layout')
+<x-mail::layout>
     {{-- Header --}}
-    @slot('header')
-        @component('mail::header', ['url' => config('app.url')])
+    <x-slot:header>
+        <x-mail::header url="config('app.url')">
             {{ config('app.name') }}
-        @endcomponent
-    @endslot
+        </x-mail::header>
+    </x-slot:header>
 
     {{-- Body --}}
     {{ $slot }}
 
     {{-- Subcopy --}}
     @isset($subcopy)
-        @slot('subcopy')
-            @component('mail::subcopy')
+        <x-slot:subcopy>
+            <x-mail::subcopy>
                 {{ $subcopy }}
-            @endcomponent
-        @endslot
+            </x-mail::subcopy>
+        </x-slot:subcopy>
     @endisset
 
     {{-- Footer --}}
-    @slot('footer')
-        @component('mail::footer')
+    <x-slot:footer>
+        <x-mail::footer>
             © {{ date('Y') }} {{ config('app.name') }}. @lang('All rights reserved.')
-        @endcomponent
-    @endslot
-@endcomponent
+        </x-mail::footer>
+    </x-slot:footer>
+</x-mail::layout>
```

Comment:
I migrated this file, but I'll admit I don't know what this file is doing and if it even makes sense - being that it is the _text_ representation.